jans-auth-server/client/src/main/java/io/jans/as/client/service/StatService.java

```diff
@@ -16,12 +16,20 @@
 public interface StatService {
     @GET
     @Produces({MediaType.APPLICATION_JSON})
-    JsonNode stat(@HeaderParam("Authorization") String authorization, @QueryParam("month") String month, @QueryParam("format") String format);
+    JsonNode stat(@HeaderParam("Authorization") String authorization,
+                  @QueryParam("month") String month,
+                  @QueryParam("start-month") String startMonth,
+                  @QueryParam("end-month") String endMonth,
+                  @QueryParam("format") String format);
 
     @GET
     String statOpenMetrics(@HeaderParam("Authorization") String authorization, @QueryParam("month") String month, @QueryParam("format") String format);
 
     @POST
     @Produces({MediaType.APPLICATION_JSON})
-    JsonNode statPost(@HeaderParam("Authorization") String authorization, @FormParam("month") String month, @FormParam("format") String format);
+    JsonNode statPost(@HeaderParam("Authorization") String authorization,
+                      @FormParam("month") String month,
+                      @QueryParam("start-month") String startMonth,
+                      @QueryParam("end-month") String endMonth,
```

Comment:
For POST it has to be FormParam, not QueryParam

Reply:
My bad. I will be more careful next time.